channels/rdpei/client/rdpei_main.c

```diff
@@ -183,6 +181,9 @@ static DWORD WINAPI rdpei_schedule_thread(LPVOID arg)
 		goto out;
 	}
 
+	RdpeiClientContext* context = (RdpeiClientContext*) rdpei->iface.pInterface;
```

Comment:
You´ll need to split declaration from assignment and put the declaration at top of scope (C support for certain compilers is really miserable, sorry)
Same for all other instances. (see the vs2010 compiler errors)

Reply:
Ok, done!